### ISSUE TITLE
Add Button to Project Popups

### ISSUE THREAD

**Issue by bld2104** [waltribeiro/cfp-mockup#2]
When a user clicks on a project image (PHLASK, Paws, etc.) a popup appears. Right now, there is a very faint "Learn More" link that takes them to the project page. 

I am proposing that we replace this link with two buttons:

Button 1 is "Learn More" and takes them to the project page
Button 2 is "Get Involved" and the action is something that directly connects them with someone on the team or the team leader (not sure what exactly this action will be yet).

**Comment by waltribeiro** [comment]
Good idea. However, do we even need modal popups? It came with the theme, but I'd be OK with using Bootstrap Cards instead. I added my sidebar "search" idea, which shows what the cards would look like. Modals don't work on mobile, for example. But I'm open to any and all ideas!
